README.md

```diff
@@ -1,2 +1,12 @@
-# UnityVRTutorialMaker
-A VR Tutorial Creator tool for Unity that lets you super easily create tutorials for how to play your game and helps users figure out the different controller elements and how to use them.  It helps with stuff like highlighting the right button / joystick on different platforms, handling vibration, has prefabs for showing tutorial tips that point to those buttons, ect... and works on all platforms automatically like Vive, Rift, Index, Quest, Go, ect..
+*VR Tutorial Maker* is an open-source, free Unity plugin created to help you quickly make controls tutorials for your VR game.
+------
+**Features:**
+- Works on all platforms (Rift, Vive, Index, Quest, Go, ...)
```

Comment:
We can move Vive, Index, Quest, and Go to **Planned Features** as those are not actually implemented yet. Currently it only works on Rift S.

Reply:
Actually Quest might already work but I haven't tested it yet